subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -57,6 +57,14 @@ The dependencies for annotation processors are not added to an IDEA module's cla
 - PMD has been updated to https://pmd.github.io/pmd-6.21.0/pmd_release_notes.html#24-january-2020---6210[PMD 6.21.0].
 - CodeNarc has been updated to https://github.com/CodeNarc/CodeNarc/blob/v1.5/CHANGELOG.md#version-15----nov-2019[CodeNarc 1.5].
 
+==== Degraded experience when using Gradle on 32-bit operating systems
```

Comment:
I think this is much too strong a title. We also don't drop support for all 32-bit OSs.

I'd focus on the visible changes, and go with a softer approach, like:

```suggestion
==== Rich console support discontinued for some 32-bit operating systems
```